.travis.yml

```diff
@@ -7,8 +7,8 @@ python:
   - '3.5'
   - '3.6'
   - '3.7'
-  # - 'pypy2.7-6.0'
-  - 'pypy3.5-6.0'
+  # - 'pypy'
+  - 'pypy3'
```

Comment:
Have you tried "pypy"?

Reply:
Tried in https://github.com/pytest-dev/py/pull/236/commits/94f8854f4b09dfde41622d3d44a199d49344aab8, same two failures (but allowed to fail, so CI is green).

Reply:
ok, let's keep it disabled then I guess.

Reply:
Removed!